dashboard/src/components/EventHeader.vue

```diff
@@ -100,4 +95,37 @@ const chapter = createDocumentResource({
   fields: ['*'],
   auto: true,
 })
+
+const isEvent = computed(() => props.form.data?.doctype === 'Event')
```

Comment:
Not `FOSS Chapter Event` ?

Reply:
<img width="890" alt="Screenshot 2024-10-09 at 5 26 56 PM" src="https://github.com/user-attachments/assets/f5f96f51-29a8-4bb2-bf99-16e731af2d2f">

I made it since it was passed as Event here - so thought not to change it, and this was once where it was used also

Reply:
Do you want me to change it to FOSS Chapter Event here ?

Reply:
This is fine then